wordpress-mobile/AztecEditor-iOS
# Use set instead of array for formatting identifiers.

This PR does a small refactor the way we return FormattingIdentifier, instead of using an array that allows duplicates we use a Set.

To test:
 - Open the demo app
 - Move the cursor around and see if the format toolbar updates correctly for the styles
